packages/nextjs/app/page.tsx

```diff
@@ -19,6 +20,77 @@ const Home: NextPage = () => {
     }
   }
 
+  const { address } = useAccount();
+
+  const { data: Ens } = useEnsName({
```

Comment:
nitpick: since `Ens` is just a variable we could have use `ens` in general we use capitalize first  letter for react component / typescript types